FHEM/00_SIGNALduino.pm

```diff
@@ -2420,7 +2422,7 @@ sub SIGNALduino_Parse_MU($$$$@)
 					}
 						
 					if (length($rawData) < ($length_min * $signal_width)) {	# Abbruch wenn Rest kleiner Mindestlaenge
-						SIGNALduino_Log3 $name, 5, "$name: $length_str last signal, aborting";
+						Log3 $name, 5, "$name: $length_str last signal, aborting" if ($dummy);
```

Comment:
Bitte wieder zurück ändern,
damit hat man keine Chance mehr es in ein eigenes Filelog zu melden und wenn das Gerät kein Dummy ist, geht es nicht.

Reply:
siehe oben